src/lang/helpers.ts

```diff
@@ -53,7 +53,7 @@ export const localeMap: { [k: string]: Partial<LanguageStrings> } = {
   tr,
   uk,
   'zh-TW': zhTW,
-  'zh': zhCN,
+  'zh-cn': zhCN,
```

Comment:
Updating this may fix the translation helper logic, but I think it breaks the language support for Obsidian unless it is just for when you have `zh-cn` selected in the Obsidian dropdown for the language value.

Reply:
The changes I made elsewhere are just to make it work, and to see the effect in obsidian, you can ignore them and just use the files I translated

Reply:
Sounds good. I will test this out later on today or tomorrow based on when I get some time.